README.md

```diff
@@ -51,6 +49,10 @@ TODO: Release when Crystal is released
 
 ## Run Examples
 
+RQT Gui:
```

Comment:
Explain this is the "master" GUI that you can load all the other plugins into or, optionally, all plugins can be launched standalone without rqt_gui also.